xcenv 1.1.0

maschall opened the pull request:
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

-----

MikeMcQuaid commented on Formula/xcenv.rb:
```diff
@@ -18,9 +18,8 @@ def install
-    store everything under Homebrew's prefix, include this in your profile:
-      export XCENV_ROOT=#{var}/xcenv
+    To enable shims and autocompletion, run this and follow the instructions:
+      xcenv init
```
Please remove these `caveats` as this is not Homebrew-specific information. Thanks!

MikeMcQuaid commented:
Looks like the `post_install` is failing: https://bot.brew.sh/job/Homebrew%20Core/9475/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install___xcenv_1_1_0_el_capitan_bottle_tar_gz/

maschall commented:
@MikeMcQuaid thanks for the reviews, I didnt realize that formula shouldn't really have post install instructions.  So I removed the caveats like you suggested and updated the website.  Thansk agin.

MikeMcQuaid commented:
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!
